### ISSUE TITLE
Question about multi-class

### ISSUE THREAD

**Issue by hahapt** [megvii-research/MOTRv2#17]
Can it support multi-class training?

**Comment by zyayoung** [comment]
Yes, but we have not checked the multi-class implementation in this repo. We will check it in a few weeks.

**Comment by GeeniTiming** [comment]
> Yes, but we have not checked the multi-class implementation in this repo. We will check it in a few weeks.

How to train on multi-class dataset?
